sml_builder/__init__.py

```diff
@@ -23,11 +27,24 @@
 import sml_builder.method  # noqa: E402
 import sml_builder.page  # noqa: E402
 import sml_builder.utils  # noqa: F401, E402
+from sml_builder.utils import get_feature_config  # noqa: E402
+from sml_builder.utils import _page_not_found, checkEmptyList  # noqa: E402
+
+content_management = get_feature_config("content_management")
 
 
 @app.route("/")
 def index():
-    return render_template("index.html")
+    # Gets the content for the home page
+    if content_management["enabled"]:
+        content = getContent("heroHomePage")
+        if checkEmptyList(content):
+            _page_not_found("heroHomePage content not found")
+
+        return render_template(
+            "index.html", content=content, cms_enabled=content_management["enabled"]
+        )
+    return render_template("index.html", cms_enabled=content_management["enabled"])
```

Comment:
Can you make this part of an else leg to be clearer that this is exercised when content_management is not enabled